src/windows/BarcodeScannerProxy.js

```diff
@@ -8,6 +8,72 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+/**
+ * @param {Function<Object>} callback
+ */
+function findCamera(callback) {
+    // Enumerate cameras and add them to the list
+    var deviceInfo = Windows.Devices.Enumeration.DeviceInformation;
+    deviceInfo.findAllAsync(Windows.Devices.Enumeration.DeviceClass.videoCapture).done(function (cameras) {
+        var camerasConfigs = cameras.map(function(camera) {
+            var cameraConfig = {
+                id: camera.id,
+                isFrontLocation: false,
+                isBackLocation: false
+            };
```

Comment:
IMO this looks too complex to create a `cameraConfig` object to use the only its property. It'd be easier to just filter out cameras, where `camera.enclosureLocation.panel === Windows.Devices.Enumeration.Panel.back` and then work with resultant array, something like 
```javascript
deviceInfo.findAllAsync(Windows.Devices.Enumeration.DeviceClass.videoCapture)
.done(function (cameras) {
    var backCameras = cameras.filter(function (camera) {
        return camera.enclosureLocation.panel === Windows.Devices.Enumeration.Panel.back;
    });

    if (backCameras.length > 0) {
        callback(backCameras[0].id);
        return;
    }
})
```